landing_page/templates/landing_page/base.html

```diff
@@ -15,7 +15,9 @@
     />
     <link rel="stylesheet" href="{% static 'landing_page/css/index.css' %}" />
     <link rel="stylesheet" href="{% static 'landing_page/css/about.css' %}" />
+    <link rel="stylesheet" href="{% static 'landing_page/css/movie.css' %}" />
     <title>ReeldIn</title>
+    <link rel="icon" type="images/x-icon" href="/landing_page/static/landing_page/images/favicon.png" />
```

Comment:
From what I can see, favicon.png is the django logo, so I think this can be removed.